src/main/java/appeng/core/Registration.java

```diff
@@ -321,7 +320,15 @@ public void registerRecipes( RegistryEvent.Register<IRecipe> event )
 		final AERecipeLoader ldr = new AERecipeLoader();
 		ldr.loadProcessingRecipes();
 	}
-
+	
+	@SubscribeEvent
+	public void registerEntities( RegistryEvent.Register<EntityEntry> event)
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') ')' is not preceded with whitespace. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.orod.org/coding_rules#rule_key=checkstyle%3Acom.puppycrawl.tools.checkstyle.checks.whitespace.ParenPadCheck)